packages/core/src/forms/studio/formula.helpers.ts

```diff
@@ -52,11 +52,19 @@ export const mapStudioFieldsWithFormula = (
   }));
 };
 
-export const getAttrsValue = (object: any) => {
+export const getAttrsValue = (object: Object, fieldType: string) => {
   if (isEmpty(object)) {
     return {};
   }
 
+  if (
+    fieldType != null ||
+    object.hasOwnProperty(fieldType) ||
+    typeof object[fieldType] !== 'string'
+  ) {
```

Comment:
This condition is a bit strange.

I think it should be better to replace it with the following. If a fieldType is given then return the parsed value of the object or else a empty object. Also, if you want to use a JSON.parse function, object type **should** be a string but that's not useful to check that here as if a fieldType is given we can suggest that the field will be a JSON field or it will make no sense. 

```
  if (fieldType != null) {
      return object[fieldType] != null ? JSON.parse(object[fieldType]) : {};
  }
```